network/database.go

```diff
@@ -27,37 +26,86 @@ type StorageInterface struct {
 	storage.InjectBase
 }
 
+// Database prefixes:
+// Processes:       network:tree/<PID>
+// DNS Requests:    network:tree/<PID>/dns/<ID>
+// IP Connections:  network:tree/<PID>/ip/<ID>
```

Comment:
Before releasing we need to make sure to NOT BREAK the UI. AFAIK this should not be a problem but we must try that first!